### PR TITLE
ci: fix permissions of PR creating action

### PR DESCRIPTION
**Pull request by consideRatio**
```yaml
    # Write permissions granted for the peter-evans/create-pull-request action
    # to push to a branch and create/update a PR
    permissions:
      contents: write
      pull-requests: write
```

Both ability to push to a branch and create/update pull requests is required for an action in the vuln-scan workflow. This PR fixes that. It also removes some inline comments that would become outdated quickly.

### CODE REVIEW

File: .github/workflows/vuln-scan.yaml [hunk 1]
```diff
@@ -171,7 +174,7 @@ jobs:
       # ref: https://github.com/jacobtomlinson/gha-find-replace
       - name: Update VULN_SCAN_TIME in Dockerfile
         if: steps.analyze.outputs.proceed == 'yes'
-        uses: jacobtomlinson/gha-find-replace@f9e200cf233bcde71011fa5e4178037881764379 # associated tag: 0.1.3
+        uses: jacobtomlinson/gha-find-replace@f9e200cf233bcde71011fa5e4178037881764379
```
Comment (manics) [inline .github/workflows/vuln-scan.yaml:177]: Is this action just doing an in-place sed?

**Review by manics — APPROVED** [review]
LGTM

File: .github/workflows/vuln-scan.yaml (hunk 1 above)
Comment (consideRatio) [reply .github/workflows/vuln-scan.yaml:177]: Yepp, it can certainly be replaced a `sed` script

Comment (consideRatio) [reply .github/workflows/vuln-scan.yaml:177]: Do you agree that we should stop using a dedicated action for this and use a `sed` script or similar instead, that doesn't involve a dedicated action that could contain a vulnerability etc?

**Pull request merged**